src/frontend/src/pages/FinancePage/FinancePage.tsx

```diff
@@ -101,6 +119,14 @@ const FinancePage = () => {
 
   return (
     <PageLayout title="Finance" headerRight={financeActionsDropdown}>
+      {isHeadOfFinance && (
+        <PendingAdvisorModal
+          open={showPendingAdvisorListModal}
```

Comment:
remove the && and j do isHeadOfFinance && showPendingAdvisorListModal

Reply:
huh?

Reply:
oh

Reply:
wait no im still confused